BeeSwiftTests/GoalViewModelTests.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  GoalViewModelTests.swift
+//  BeeSwiftTests
+//
+//  Created by krugerk on 2024-11-25.
+//
```

Comment:
I've been defaulting to just completely leaving out these header blocks. It's all in the git metadata anyway. Probably there's a project setting I should go change to change the default.